python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -177,89 +176,10 @@ def __init__(
         if workers is None:
             workers = self.client.has_what().keys()  # Default to all workers
         self.workers = workers
-
```

Comment:
I think it would clean up the regressor/classifier dask wrappers even further if the constructor params that are proxied down to the underlying single GPU model were specified using `kwargs`, rather than having to duplicate them in the constructor. For features are are not supported, perhaps a validation function can be used to throw a warning when unsupported options are set